doc/user_manual.tex

```diff
@@ -135,14 +140,33 @@ \section{FeedForwardNeuralNetwork}
 \end{equation}
 where $\actf: \mathbb{R} \rightarrow \mathbb{R}$ is a so-called \emph{activation function} and we have made use of the Einstein notation, for which repeated symbols on different levels imply a summation, e.g. $x_i^j y^i = \sum_i x_i^j y^i$.
 The activation function is required to account for non-linear effects.
-The FFNN library provides three activation functions by default, but the user has the possibility to define and use its own, even though its storage on a file it is not supported (explained later).
-These three activation functions are:
+The FFNN library provides several well-known activation functions by default, but the user has the possibility to define and use its own, even though its storage on a file it is not supported (explained later).
+These available activation functions are:
 \begin{enumerate}
-\item the \emph{logistic function}: $\logaf(x) \equiv \frac{1}{1+e^{-x}}$;
-\item the \emph{gaussian}: $\gssaf(x) \equiv e^{-x^2}$;
-\item the \emph{identity}: $\idaf(x) \equiv x$. This function is linear and therefore should not be used normally, but it can be useful in some special circumstances.
+\item the \emph{logistic function}: $\logaf(x) \equiv \frac{1}{1+e^{-x}}$ \\
+The current default for hidden and output units.
+\item the \emph{tansig function}: $\tansaf(x) \equiv \frac{2}{1+e^{-2*x}} - 1$ \\
+Faster approximation to the regular tanh. Similar to logistic function, but with output in [-1,1].  
+\item the \emph{gaussian}: $\gssaf(x) \equiv e^{-x^2}$ \\
+Unlike normal activation functions it is symmetric around origin and not monotonic. Still there are use cases for it.
+\item the \emph{sine function}: $\sinaf(x) \equiv sin(x)$ \\
+Periodic activation function for special use cases.
+\item the \emph{identity}: $\idaf(x) \equiv x$ \\
+This function is linear and therefore should not be used for (all) hidden units.
+\item the \emph{PReLU\footnote{\href{https://arxiv.org/abs/1502.01852}{arXiv:1502.01852}} function}: $\reluaf(x) \equiv 
+\begin{cases}
+  x & \text{if $x>0$} \\
+  \alpha * x & \text{if $x<=0$}
```

Comment:
ahahah I think you are coding way too much since too long :)
the * should be omitted, or replaced by a dot ;)